doc/source/release-notes/Percona-XtraDB-Cluster-5.7.14-26.17.rst

```diff
@@ -0,0 +1,100 @@
+.. rn:: 5.7.14-26.17
+
+=====================================
+|Percona XtraDB Cluster| 5.7.14-26.17
+=====================================
+
+.. note:: This release is dedicated to the memory of Federico Goncalvez,
+   our colleague with Percona's Uruguayan team
+   until his tragic death on September 6, 2016.
+
+   Fede, you are missed.
+
+Percona is glad to announce the release of
+|PXC| 5.7.14-26.17 on September 26, 2016.
+Binaries are available from the
+`downloads area
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-57/LATEST>`_
+or from our :ref:`software repositories <install>`.
+
+|PXC| 5.7.14-26.17 is the first GA release in the 5.7 series,
+based on the following:
+
+* `Percona Server 5.7.14 <http://www.percona.com/doc/percona-server/5.7/release-notes/Percona-Server-5.7.14.html>`_
+
```

Comment:
Bad version/link, should be: 5.7.14-8 https://www.percona.com/doc/percona-server/5.7/release-notes/Percona-Server-5.7.14-8.html